parsl/tests/site_tests/README.rst

```diff
@@ -0,0 +1,60 @@
+Site Testing
+============
+
+This doc will cover two items:
+
+1. Running the site tests on a site
+2. How to prep a new site for site testing
+
+
+The site tests are setup with the assumption that there is a conda installation
+available either through modules available on the system, or from a user's personal
+installation.
+
+* `conda_setup.sh` will create a unique conda env mapped to `parsl/parsl_<GIT_HASH>` and
+  leave a script `~/setup_parsl_test_env.sh` that will activate this env.
+
+* `site_config_selector.py` is a facade that checks the site's hostname and picks a matching
+  configuration from `parsl/parsl/tests/configs`.
+
+* The tests in this folder are designed to use the site_config_selector and runs test against
+  that config.
+
+
+Running tests
+-------------
+
+Assuming the site tests are supported on your site, here's how you run tests::
+
+  1. Checkout the git tag/hash for testing
+     * If you are running tests on a system where the $HOME is not mounted on compute nodes
+       checkout to $WORK / $SCRATCH or the equivalent. For eg, Comet and Stampede2
+
+  2. `cd parsl/tests/site_tests`
+  3. `bash conda_setup.sh`
+  4. Now check if the setup script is available at `~/setup_parsl_test_env.sh`
+  5. Go back to the parsl root dir
+  6. Run tests with `make site_test`
```

Comment:
Can I run steps 1 and 6, ignoring steps 2-5? If not, please script steps 2-5 so I don't have to do them explicitly.

Reply:
That'll be in round 2.